### PR TITLE
C++: rename SLINT_FEATURE_STD in SLINT_FEATURE_FREESTANDING with the opposite meaning

### PR DESCRIPTION
**Pull request by ogoffart**

### CODE REVIEW

File: api/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -129,31 +129,33 @@ define_renderer_winit_compat_option(skia-opengl)
...
 
-define_cargo_dependent_feature(gettext "Enable support of translations using gettext" OFF SLINT_FEATURE_STD)
-define_cargo_dependent_feature(accessibility "Enable integration with operating system provided accessibility APIs" ON SLINT_FEATURE_STD)
+define_cargo_dependent_feature(gettext "Enable support of translations using gettext" OFF "NOT SLINT_FEATURE_FREESTANDING")
+define_cargo_dependent_feature(accessibility "Enable integration with operating system provided accessibility APIs" ON "NOT SLINT_FEATURE_FREESTANDING")
 
-if (NOT SLINT_FEATURE_STD)
-    list(APPEND features i-slint-core/libm i-slint-core/unsafe-single-threaded)
+if (SLINT_FEATURE_FREESTANDING)
+    list(APPEND features)
```
Comment (tronical) [inline api/cpp/CMakeLists.txt:152]:
```suggestion
    list(APPEND features freestanding)
```

**Review by tronical — APPROVED** [review]
Looks good to me, but I think there is one small bug - see comment.

File: api/cpp/CMakeLists.txt (hunk 1 above)
Comment (ogoffart) [reply api/cpp/CMakeLists.txt]: Actually not, this line should be removed.

File: api/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -129,31 +129,33 @@ define_renderer_winit_compat_option(skia-opengl)
...
 
-define_cargo_dependent_feature(gettext "Enable support of translations using gettext" OFF SLINT_FEATURE_STD)
-define_cargo_dependent_feature(accessibility "Enable integration with operating system provided accessibility APIs" ON SLINT_FEATURE_STD)
+define_cargo_dependent_feature(gettext "Enable support of translations using gettext" OFF "NOT SLINT_FEATURE_FREESTANDING")
+define_cargo_dependent_feature(accessibility "Enable integration with operating system provided accessibility APIs" ON "NOT SLINT_FEATURE_FREESTANDING")
 
-if (NOT SLINT_FEATURE_STD)
-    list(APPEND features i-slint-core/libm i-slint-core/unsafe-single-threaded)
+if (SLINT_FEATURE_FREESTANDING)
+    list(APPEND features freestanding)
```
Comment (ogoffart) [inline api/cpp/CMakeLists.txt:152]:
```suggestion
```

**Pull request merged**